bot/src/infrastructure/http/zkill.ts

```
@@ -5,126 +1,292 @@
+   * @returns Array of killmails for the character
+   *
+   * Note: The API returns kills in reverse chronological order (newest first).
+   * Each page typically contains 100 killmails.
+   *
+   * Example URL: https://zkillboard.com/api/kills/characterID/268946627/page/1/
    */
-  async getCharacterKills(
+  public async getCharacterKills(
     characterId: number,
-    limit: number = 50
+    page: number = 1
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)